src/main/java/io/localmotion/smokefreeplaygrounds/command/RecordPlaygroundObservationCommand.java

```diff
@@ -13,6 +18,10 @@
     String initiativeId;
     String observer;
     Boolean smokefree;
+
+    @NotBlank(message = "Message must have at least " + SmokefreeConstants.PlaygroundObservation.MINIMUM_COMMENT_LENGTH + " character")
```

Comment:
this validation should be removed as the user is not obliged to enter a comment

Reply:
Agree. I removed it